src/lib/transaction_database/intf.ml

```diff
@@ -0,0 +1,34 @@
+open Coda_base
+open Signature_lib
+
```

Comment:
it may also be nice to include a top comment describing the purpose of the module and what it's doing at a high level